masterlist.yaml

```diff
@@ -7889,3 +7889,14 @@ plugins:
         crc: 0x82DAC22D
         util: '[FNVEdit v4.0.4](https://www.nexusmods.com/newvegas/mods/34703)'
         itm: 1
+
+  - name: 'HiddenValleyOverhauled.esm'
+    url: [ 'https://www.nexusmods.com/newvegas/mods/70859/' ]
+    dirty:
+      - <<: *quickClean
+        crc: 0xA2898CFC
+        util: '[FNVEdit v4.0.4](https://www.nexusmods.com/newvegas/mods/347003)'
```

Comment:
It would be ideal to use the latest version of xEdit available at the time of providing such data.
Currently that is xEdit v4.1.5k, which can be downloaded from xEdit's Discord (from within the `xedit-builds` channel).

Reply:
CRC is indeed the same in both versions, only change is absence of NexusMods link in 4.1.5k output.